README.md

```diff
@@ -1,60 +1,76 @@
+> An [`abstract-leveldown`](https://github.com/Level/abstract-leveldown) compliant store on top of [IndexedDB](https://developer.mozilla.org/en-US/docs/Web/API/IndexedDB_API), which is in turn implemented on top of [LevelDB](https://github.com/google/leveldb) which brings this whole shebang full circle.
 
-**Most people use [levelup](http://github.com/rvagg/node-levelup) on top of this library. See `test-levelup.js` for details**
+[![level badge][level-badge]](https://github.com/level/awesome)
+[![npm](https://img.shields.io/npm/v/level-js.svg)](https://www.npmjs.com/package/level-js)
+[![npm next](https://img.shields.io/npm/v/level-js/next.svg)](https://www.npmjs.com/package/level-js)
+[![Travis](https://secure.travis-ci.org/Level/level.js.svg?branch=master)](http://travis-ci.org/Level/level.js)
+[![npm](https://img.shields.io/npm/dm/level-js.svg)](https://www.npmjs.com/package/level-js)
 
-For some demos of it working, see @brycebaril's presentation "Path of the NodeBases Jedi": http://brycebaril.github.io/nodebase_jedi/#/vanilla
+## Background
 
-[![NPM](https://nodei.co/npm/level-js.png)](https://nodei.co/npm/level-js/)
+Here are the goals of `level-js`:
 
-level.js uses [IDBWrapper](https://github.com/jensarps/IDBWrapper) by jensarps to ensure compatibility between IDB implementations.
+- Store large amounts of data in modern browsers
+- Pass the full `abstract-leveldown` test suite
+- Support [Buffer](https://nodejs.org/api/buffer.html) values (in all target environments)
+- Support all key types of IndexedDB Second Edition, including binary keys (depends on environment)
+- Support all value types of the structured clone algorithm (depends on environment)
+- Be as fast as possible
+- Sync with [multilevel](https://github.com/juliangruber/multilevel) over either ASCII or binary transports.
 
-Here are the goals of this level.js:
+Being `abstract-leveldown` compliant means you can use many of the [Level modules](https://github.com/Level/awesome/) on top of this library. For some demos of it working, see @brycebaril's presentation [Path of the NodeBases Jedi](http://brycebaril.github.io/nodebase_jedi/#/vanilla).
 
-- Store large amounts of ascii (strings, JSON) and binary (Buffer) data in modern browsers
-- Be as fast as possible
-- Use the leveldown test suite and sync with [multilevel](https://github.com/juliangruber/multilevel) over either ascii or binary transports (websockets and xhr both have ascii/binary modes in browsers now)
+## Example
 
-Being leveldown compatible means you can use many of the [level-* modules](https://github.com/rvagg/node-levelup/wiki/Modules) on top of this library.
+**This assumes use of version `3.0.0-rc1`. The next release will have an upgrade guide.**
 
-## install
+```js
+var levelup = require('levelup')
+var leveljs = require('level-js')
+var db = levelup(leveljs('bigdata'))
 
-```sh
-npm install level-js
-```
+db.open(function (err) {
```

Comment:
Oh, with `levelup`, we don't need explicit open.

Reply:
Correct.